lib/theme.json

```diff
@@ -341,7 +352,7 @@
 							}
 						},
 						{
-							"selector": " > *",
+							"selector": " > :where(*)",
```

Comment:
After the rebase, do we need to update the `grid` layout type to also use this specificity?

Reply:
Actually, looking better at it, we don't! the reason being the `*` selector has 0 specificity already. I'm not sure why I wrapped it in a `:where` initially 🤔 . Will update for consistency!